src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -36,6 +37,15 @@ public static List<HearingChannel> getHearingChannels(@Valid SscsCaseData caseDa
     }
 
     public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
+
+        if (caseData.getAdjournCaseTypeOfNextHearing() != null) {
```

Comment:
Also include the feature flag enabled check

Reply:
This will require getHearingChannel to take referenceServiceData as an additional arg

Reply:
added feature flag